xcape/core.py

```diff
@@ -84,7 +84,7 @@ def _cape_dummy(p, t, td, ps, ts, tds, pres_lev_pos,
 def _any_dask_array(*args):
     return any([isinstance(a, da.Array) for a in args])
 
-def calc_cape(p, t, td, ps, ts, tds, **kwargs):
+def calc_cape(p, t, td, ps, ts, tds, *args, **kwargs):
```

Comment:
added pres_lev_pos as an optional argument in the API. It will be fixed to =1 if vertical lev is sigma, or it will expect an array like pressure if vertical_lev is pressure.

Reply:
I like this solution!